examples/profile.js

```diff
@@ -0,0 +1,52 @@
+goog.provide('profile_example');
+
+
+goog.require('ngeo');
+goog.require('ngeo.profileDirective');
+
+
+/** @const **/
+var app = {};
+
+
+/** @type {!angular.Module} **/
+app.module = angular.module('app', ['ngeo']);
+
+
+
+/**
+ * @constructor
+ * @param {angular.$http} $http The $http angular service.
+ * @param {angular.Scope} $scope The $scope angular service.
+ */
+app.MainController = function($http, $scope) {
+
+  $http.get('data/profile.json').then(angular.bind(this, function(resp) {
+    this['profileData'] = resp.data;
+  }));
+
+  this['point'] = null;
+
+  this.scope_ = $scope;
+};
+
+
+/**
+ * @param {Object} point The point object with x, y, elevation, distance
+ * information.
```

Comment:
It's actually an item in the data array given to the profile.
What makes you think it's not the case?


On Fri, Feb 27, 2015 at 3:03 PM, Éric Lemoine <notifications@github.com>
wrote:

> In examples/profile.js
> <https://github.com/camptocamp/ngeo/pull/173#discussion_r25507676>:
>
> > + */
> > +app.MainController = function($http, $scope) {
> > +
> > +  $http.get('data/profile.json').then(angular.bind(this, function(resp) {
> > +    this['profileData'] = resp.data;
> > +  }));
> > +
> > +  this['point'] = null;
> > +
> > +  this.scope_ = $scope;
> > +};
> > +
> > +
> > +/**
> > + * @param {Object} point The point object with x, y, elevation, distance
> > + * information.
>
>  The object sent to callbacks is supposed to be the point as it was given
> to the profile. It's the raw data.
> Are you sure that is the case currently?
> … <#14bcb59076b834f7_>
> -- Eric Lemoine Camptocamp France SAS Savoie Technolac, BP 352 73377 Le
> Bourget du Lac, Cedex Tel : 00 33 4 79 44 44 94 Mail :
> eric.lemoine@camptocamp.com http://www.camptocamp.com
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/camptocamp/ngeo/pull/173/files#r25507676>.
>



-- 
Pierre GIRAUD
Géomaticien, Analyste

Camptocamp France SAS
Savoie Technolac, BP 352
73377 Le Bourget du Lac, Cedex

Tel : 00 33 4 79 44 44 93
Mail : pierre.giraud@camptocamp.com
http://www.camptocamp.com